app/styles/app.scss

```diff
@@ -13,6 +13,7 @@
 @import 'components/x-card';
 
 $primary-color-palette: $color-orange;
+$locked-breakpoint: 5000px;
```

Comment:
ah yeah i realized that after i was laying in bed last night. nice find and many thanks to ember-paper (@miguelcobain) for this kind of ease of development